### ISSUE TITLE
Deprecated rnpm for React Native 0.60.3

### ISSUE THREAD

**Issue by Bruno-Furtado** [mockingbot/react-native-zip-archive#153]
I recently upgraded the version of React Native to 0.60.3 and I'm getting the following alert:

```
warn The following packages use deprecated "rnpm" config that will stop working from next release:
  - react-native-zip-archive: https://github.com/mockingbot/react-native-zip-archive
Please notify their maintainers about it. You can find more details at https://github.com/react-native-community/cli/blob/master/docs/configuration.md#migration-guide.
```

Here is my env:

```
React Native: version:
"react": "16.8.6"
"react-native": "0.60.3"

Dependency version:
"react-native-zip-archive": "4.1.1"
```

Could you release a version with this improvement?

**Comment by plrthink** [comment]
Sure, actually I'm working on the new version to adopt for React Native 0.60, which would be released at next week maybe.

**Comment by mars-lan** [comment]
Any update @plrthink? Thanks.

**Comment by plrthink** [comment]
@Bruno-Furtado @mars-lan Just published the alpha version 5.0.0 to test, could you please try that and if it works I would release the updates ASAP.

**Comment by plrthink** [comment]
Sorry for not mentioned, you should install the alpha version via '5.0.0-alpha.0'.

**Comment by mars-lan** [comment]
The warning message is gone using `5.0.0-alpha.0`. Thanks, @plrthink.

**Comment by mars-lan** [comment]
Ping @plrthink could you cut a release for 5.0.0?

**Comment by plrthink** [comment]
@mars-lan Actually I'm planning for taking a few days before the release since it introduces the breaking change. And I'm still gathering feedback about the alpha version to ensure it works well and draft the new doc to reflect the changes. It would be longer than a couple of days I suppose.

**Comment by mars-lan** [comment]
Makes sense. Will await eagerly :)

**Comment by plrthink** [comment]
@mars-lan and you can track the progress in this PR https://github.com/mockingbot/react-native-zip-archive/pull/155.

**Comment by plrthink** [comment]
closed since the fixes have been released via 5.0.0
